src/pages/Version.tsx

```diff
@@ -97,7 +99,12 @@ export const VersionPage: React.FC = () => {
         getVersion();
       }
     }
-  }, [patchData, getVersion, id]);
+    // if there was an error fetching the patch and the patch id was a real id it could be a periodic build
+    // in which case we should check if theres a corresponding version associated with it.
+    if (validatePatchId(id) && patchError) {
```

Comment:
The console is logging lots of (expected) errors, which we should probably suppress. But moreover, I feel like this approach is not ideal because we'd start logging all of these GraphQL errors to Splunk, and since they're not really errors, it would be noisy and misleading. Is there a way that the resolvers could be updated, or a new one added, to avoid relying on an error?

Also interesting because this is really similar to the repo/project issue we spoke about the other day! This may be a pattern we start seeing more often.